cmd/editorconfig-checker/main.go

```diff
@@ -60,6 +60,9 @@ func init() {
 		configPaths = append(configPaths, defaultConfigFileNames[:]...)
 	} else {
 		configPaths = append(configPaths, configFilePath)
+		if configFilePath == ".ecrc" {
+			logger.Warning("The default configuration file name `.ecrc` is deprecated. Use `.editorconfig-checker.json` instead.")
+		}
 	}
 
 	currentConfig, _ = config.NewConfig(configPaths)
```

Comment:
A better test could be to check here for the currentConfig file name to contain `.ecrc` only if comfigFilePath is empty, then display the warning message